### DISCUSSION TITLE
Request

### DISCUSSION THREAD

**Discussion by AnitanshMishra** [WirelessAlien/BhagavadGitaApp#6] [General]
Please make an app of tulsidas Ram Charit Manas. 🙏🏻🙏🏻🙏🏻

**Comment by WirelessAlien** [comment]
> Please make an app of tulsidas Ram Charit Manas. 🙏🏻🙏🏻🙏🏻

There's no need to create a separate app for Ram Charit Manas, we can integrate it into this one. It would have been easier if there was a dataset available for Ram Charit Manas, but there isn't, so it will take time to do that.
